### ISSUE TITLE
travis: cobertura error while parsing the java file SpringZeebeBroker.java

### ISSUE THREAD

**Issue by jangalinski** [zeebe-io/spring-zeebe#27]
```
[INFO] --- cobertura-maven-plugin:2.7:cobertura (default-cli) @ spring-zeebe-broker ---

[INFO] Cobertura 2.1.1 - GNU GPL License (NO WARRANTY) - See COPYRIGHT file

[INFO] Cobertura: Loaded information on 3 classes.

[WARN] JavaNCSS got an error while parsing the java file /home/travis/build/zeebe-io/spring-zeebe/broker/spring-zeebe-broker/src/main/java/io/zeebe/spring/broker/config/SpringZeebeBroker.java

ParseException in STDIN

Last useful checkpoint: "io.zeebe.spring.broker.config.SpringZeebeBroker.stop()"

Encountered " ")" ") "" at line 54, column 20.

Was expecting one of:
```

**Comment by jangalinski** [comment]
cobertura issue due to not supported lambda: https://github.com/cobertura/cobertura/issues/176

**Comment by jangalinski** [comment]
switch to jacoco

**Comment by jangalinski** [comment]
solved with 7eb0176
